viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1041,16 +1047,29 @@ export class Cognite3DViewer {
       renderController.update();
       this._revealManager.update(this.camera);
 
-      if (renderController.needsRedraw || this._revealManager.needsRedraw || this._slicingNeedsUpdate) {
+      if (
+        (renderController.needsRedraw || this._revealManager.needsRedraw || this._slicingNeedsUpdate) &&
+        !this.shouldSuspendRendering()
+      ) {
         this.triggerUpdateCameraNearAndFar();
         this._revealManager.render(this.renderer, this.camera, this.scene);
         renderController.clearNeedsRedraw();
         this._revealManager.resetRedraw();
         this._slicingNeedsUpdate = false;
+        this._lastFrameTimestamp = performance.now();
```

Comment:
@larsmoa  You probably don't need performance.now in render loop. It has faster alternatives (but less precise though). 

Check it out
* https://jsbench.me/k3kh4nni14/1
* https://stackoverflow.com/questions/30795525/performance-now-vs-date-now

Reply:
Oh, didnt know - thx. Fixed.